src/polyfill/codec.ts

```diff
@@ -0,0 +1,11 @@
+; (function() {
+    // TODO: TextEncoder
+    if (typeof TextEncoder === undefined) {
```

Comment:
this code is always false
```suggestion
    if (typeof TextEncoder === 'undefined') {
```